pallets/utxo/src/lib.rs

```diff
@@ -640,6 +787,9 @@ pub mod pallet {
                         crate::script::verify(&tx, &input_utxos, index as u64, witness, lock)
                             .map_err(|_| "script verification failed")?;
                     }
+                    _ => {
```

Comment:
Minor: Please do not use `_` at all unless we must. I understand you're doing this as placeholder, but preferably, add the arms that matches your enum and add the TODO notes there. We don't want to do the mistake of forgetting this and then failing to treat new types of outputs.